### PR TITLE
fix: listen to `SKIPPED` tasks on "zaak" and "taak" processing

### PR DESCRIPTION
**Pull request by xiduzo**
- [feat(BatchProcessService): allow option for passing a timeout](https://github.com/infonl/dimpact-zaakafhandelcomponent/commit/cf8f961d1a9de0fe2099ce0b11048d0e6b0ed0f3)
- [fix(Zaken,Taken): listen to ANY optcode for progression](https://github.com/infonl/dimpact-zaakafhandelcomponent/commit/76b4ae4a56d66832a993351735e15748688cace7)
- [chore(ZakenVrijgevenDialog): proper use of *ngIf](https://github.com/infonl/dimpact-zaakafhandelcomponent/commit/62585a7feb4de37ca183feb960d3d0444cb29d9d)

Closes [PZ-3977](https://dimpact.atlassian.net/browse/PZ-3977)

### CODE REVIEW

File: src/main/app/src/app/taken/taken-werkvoorraad/taken-werkvoorraad.component.ts [hunk 1]
```diff
@@ -7,8 +7,8 @@ import {
   AfterViewInit,
   Component,
   OnInit,
-  ViewChild,
   signal,
+  ViewChild,
```
Comment (bas-info-nl) [inline src/main/app/src/app/taken/taken-werkvoorraad/taken-werkvoorraad.component.ts:11]: Is this order valid with our lint settings? Please check by running `./gradlew check`

File: src/main/app/src/app/zaken/zaken-werkvoorraad/zaken-werkvoorraad.component.ts [hunk 2]
```diff
@@ -259,9 +259,11 @@ export class ZakenWerkvoorraadComponent
     this.batchProcessService.start({
       ids: zaken.map(({ id }) => id),
       progressSubscription: {
-        opcode: Opcode.UPDATED,
+        opcode: Opcode.ANY,
         objectType: ObjectType.ZAAK_ROLLEN,
-        onNotification: (id) => {
+        onNotification: (id, event) => {
+          if (event.opcode !== Opcode.UPDATED) return;
+
```
Comment (bas-info-nl) [inline src/main/app/src/app/zaken/zaken-werkvoorraad/zaken-werkvoorraad.component.ts:266]: Is the following logic the same for `Opcode.DELETED` and `Opcode.UPDATED` ?

**Review by bas-info-nl — APPROVED** [review]
Looks good, just some minor questions.

**Review by edgarvonk — APPROVED** [review]
Nice!

One small thing: our convention in PR descriptions is to end them with one paragraph where we link it to the PR. This so we see the link to the JIRA issues in the release notes.
See: https://github.com/infonl/dimpact-zaakafhandelcomponent/blob/main/CONTRIBUTING.md

**Comment by edgarvonk** [comment]
Oh, one question: the title mentions `listen to SKIPPED tasks on "zaak" and "taak" processing` but the story is _only_ about zaken, not taken! That's a different story.

File: src/main/app/src/app/taken/taken-werkvoorraad/taken-werkvoorraad.component.ts (hunk 1 above)
Comment (xiduzo) [reply src/main/app/src/app/taken/taken-werkvoorraad/taken-werkvoorraad.component.ts:11]: gradlew says 👌

**Pull request merged**